### PR TITLE
Added update error message

### PR DESCRIPTION
**Pull request by antoniomdk**

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Please always reference the issue if it has something to do with it. What did you update? Maybe that could have been in the commit message. Now, I can only guess or must look at the source code.

**Comment by niccokunzmann** [comment]
Thanks for the apk.

**Comment by antoniomdk** [comment]
Ok, sorry. If threre is any problem with, tell me.

**Comment by antoniomdk** [comment]
I have added a progress bar and some message.

File: FlappySVG_Android/app/src/main/java/fossasia/flappysvg/GameActivity.java [hunk 1]
```diff
@@ -66,34 +69,46 @@ public boolean onCreateOptionsMenu(Menu menu) {
     }
 
     void LoadGame(){
-        //Checking network connection. If there is connection, it downloads the game to use later locally.
-        if(isNetworkAvailable()){
-            myWebView.loadUrl(game_web_url);
-            new DownloadTask().execute(download_file_path);
+        if (isThereAPreviousVersion()) {
+            Log.d("Loading Game", local_game_path);
+            myWebView.loadUrl("file:///" + local_game_path);
```
Comment (niccokunzmann) [inline FlappySVG_Android/app/src/main/java/fossasia/flappysvg/GameActivity.java]: Updating while loading the files form the web view creates possible race-conditions.

**Comment by niccokunzmann** [comment]
I can still understand the code. It looks like it would do as expected. I will test the app as soon as I get the device back.

File: FlappySVG_Android/app/src/main/java/fossasia/flappysvg/GameActivity.java (hunk 1 above)
Comment (antoniomdk) [inline FlappySVG_Android/app/src/main/java/fossasia/flappysvg/GameActivity.java]: Actually, that code is deleted. The app checks if there is local game, and if it is true, the app load it. If it is not true, the app download the game. And if you aare playing and you update the game, the app reload the webview.

**Comment by antoniomdk** [comment]
Perfect!. I am working in the description but it is not ready yet. I have a question. How Do I have to orient the description? as an open source game o as a GCI project?

**Comment by niccokunzmann** [comment]
I would say it is an open source project, which is true, see license and open code. I would not use the word but rather describe what it means, since the audience may not be familiar with this term. 
Something like: You can create own characters and levels and there is guidance by a community. People developed this game who are just like you. 
- Something in this sense.
You can write about google code-in. The objective - I would propose - is to create appetite to try the game and start developing it.

**Comment by antoniomdk** [comment]
Perfect, I am going to use that. Thanks.

**Comment by niccokunzmann** [comment]
I tested it and it works. Very good.

**Comment by antoniomdk** [comment]
Perfect. If there is any problem, tell me.

**Pull request merged**